hwdetect/hwdetect.sh

```diff
@@ -0,0 +1,71 @@
+#!/bin/bash
+
+USB_CVT=1
+BLK_DEV=2
+OTHER=3
+
+STATUS_OK=0
+
+function detrDevice
+{
+	local dev=$1
+
+	if [[ "$dev" == *"ttyUSB"* ]]
+	then
+		return $USB_CVT
+	fi
+
+	if [[ "$dev" == *"sd"? ]]
+	then
+		return $BLK_DEV
+	fi
+
+	return $OTHER
+}
```

Comment:
Is it possible to exclude this function and use just device name without enumeration?

Reply:
Yes, then `printInfo` will look like this :
```
function printInfo
{
	IFS=$'\n'
	cur=${1}
	
	for dev in $cur
	do
		echo $dev

		case $dev in
			*ttyUSB*)
			       	echo "Usb to ttl convertor"
				;;
			*sda?)
			       	echo "Block device"
					sudo lshw -short | grep ${dev//[<> ]} -m 1
				;;
			*)
			       	continue
				;;
		esac

	done
}
```

In this case it's better, I think. But in a case, where are 2+ section depended on device type trick with a numeration looks better. 
And probably there are cases, where only name of device is not enough and having function with device type identification is must-have.

Should I update the code?